src/NHibernate/Driver/Sql2008ClientDriver.cs

```diff
@@ -5,7 +5,13 @@
 
```

Comment:
For this PR to be a complete preparation to .NetStandard, shouldn't this driver be switched to reflection based when compiled inside NHibernate.csproj, and so cease using directly `System.Data.SqlClient`?